Misc/NEWS.d/next/Core and Builtins/2017-09-20-18-28-09.bpo-31530.CdLOM7.rst

```diff
@@ -0,0 +1 @@
+Fixed crashes when iterating over a file on multiple threads.
```

Comment:
I suggest to complete the description:

seek() and next() methods of file objects now raise an exception during concurrent operation on the same file object. A lock can be used to prevent the error.

Reply:
done